frontend/src/features/splReports/hooks/useProjectSnapshotApi.ts

```diff
@@ -87,9 +87,10 @@ const snapshotToasts: LifecycleToasts = {
  */
 export const useProjectSnapshotApi = () => {
   const dispatch = useDispatch();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   const defaultAxios = useCallback(
     getAxios(dispatch, { errorToast: () => toast.error('Failed to load reports') }),
-    [],
+    [dispatch],
```

Comment:
dependency resolved, but couldn't figure out how to resolve the anonymous function in the callback.